### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by Timthreetwelve** [Timthreetwelve/GetMyIP#117] [Announcements]
As the Thanksgiving holiday nears here in the US, I want to give thanks for all of the people out there that have downloaded Get My IP, and to those that help keep the project going by testing and by contributing language translations.

**THANK YOU!** 🙏🏻❤️

**Comment by CMTriX** [comment]
Thank **you** for this amazing tool!  Happy holidays, fijne feestdagen!
